bloom/filter_test.go

```diff
@@ -95,6 +95,68 @@ func TestFilterInsert(t *testing.T) {
 	}
 }
 
+// TestFilterFPRange checks that new filters made with out of range
+// false positive targets result in either max or min false positive rates.
+func TestFilterFPRange(t *testing.T) {
```

Comment:
Actually, I'll go ahead and refactor this along with the mentioned thing and paste the refactored version here in as it'll only take a few minutes.